index.html

```diff
@@ -55,6 +54,227 @@ <h2 class="green">I’m a Full Stack developer from Microverse</h2>
+        
+      </div>
+
+      <!-- Project 5 -->
+
+      <div class="project">
+        <div class="project-image">
+          <img src="" alt="" placeholder="">
+        </div>
+        <div class="sections-title">
+          <h3>
+            Application deploy
+          </h3>
+        </div>
+        <ul class="project-built-with">
+          <li><a href="">Ruby on Rails</a></li>
+          <li><a href="">CSS</a></li>
+          <li><a href="">Javascript</a></li>
+          <li><a href="">HTML</a></li>
+        </ul>
+
+        <div class="project-buttons">
+          <a href="">See project</a>
+        </div>
+        
+      </div>
+
+    </div>
+  </section>
+
+  <section class="about-myself">
+
+    <header>
+      <h2 class="green">
+        About me
+      </h2>
+      <hr class="green">
+    </header>
+
+    <p>
+        I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+        don’t hesitate to contact me.
+    </p>
+
+    <div class="project-buttons see-resume-button">
+      <a href="">Get my Resume</a>
+    </div>
+    
+    <div class="competences">
+      
+      <div class="languages">
+        <div class="competences-image">
+          <img src="images/Icons/languages.png" alt="languages" placeholder="">
+        </div>
+        <div class="sections-title">
+          <h3>
+            Languages
+          </h3>
+        </div>
+        <ul class="list-competences">
+          <li><a href="">Javascript</a></li>
+          <li><a href="">Ruby</a></li>
+          <li><a href="">HTML</a></li>
+          <li><a href="">CSS</a></li>
+        </ul>
+    
+      </div>
+
+
+      <div class="frameworks">
+        <div class="competences-image">
+          <img src="images/Icons/frameworks.png" alt="languages" placeholder="">
+        </div>
+        <div class="sections-title">
+          <h3>
+            Frameworks
+          </h3>
+        </div>
+        <ul class="list-competences">
+          <li class="competences-item"><a href="">React.js</a></li>
+          <li class="competences-item"><a href="">Ruby on Rails</a></li>
+          <li class="competences-item"><a href="">RSpec</a></li>
+          <li class="competences-item"><a href="">Capybara</a></li>
+          <li class="competences-item"><a href="">Selenuim</a></li>
+        </ul>
+    
+      </div>
+
+      <div class="skills">
+        <div class="competences-image">
+          <img src="images/Icons/skills.png" alt="languages" placeholder="">
+        </div>
+        <div class="sections-title">
+          <h3>
+            Skills
+          </h3>
+        </div>
+        <ul class="list-competences">
+          <li><a href="">Database management</a></li>
+          <li><a href="">version control</a></li>
+          <li><a href="">CLI</a></li>
+          <li><a href="">Web development</a></li>
+          <li><a href="">API design</a></li>
+        </ul>
```

Comment:
- [ ] Kindly try to match your design to the Figma design. The section has the margins (top, bottom, left, right) defined in the design. The section uses the font and colors (background and font) defined for the template. :+1: 

Your design             |  Figma Design
:-------------------------:|:-------------------------:
![yourdesign_myself](https://user-images.githubusercontent.com/70483835/227732520-944fd1de-bef4-4266-b5fc-ac0d4e985ecd.png) | ![figma_myself](https://user-images.githubusercontent.com/70483835/227732516-49f176a5-e9cd-47bb-add1-5cbd04bc1329.png)